app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,19 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag sign_in_path, class: "sign_in" do
+        %header
+          %h1 Sign in
+        %fieldset
+          .form-group
+            %label Email Address
+            .row
+              .col-sm-4
+                = email_field_tag 'email', nil, class: "form-control"
+          .form-group
+            %label Password
+            .row
+              .col-sm-4
+                = password_field_tag 'password', nil, class: "form-control"
+        %fieldset.form-group.action
+          %input(type="submit" value="Sign in" class="btn btn-default")
```

Comment:
HAML simplifies the template, so I just use haml's markup, maybe should use Rails helper as much possible as I can?